data/covering_grammar/lib/make_test_file.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+
+"""Makes test file.
+
+Takes the gold data and the model output, and creates a three-column TSV where
+each line has a word, its gold pronunciation, and the predicted pronunciation.
+Assumes that the input files have the same words in the same order.
+"""
+
+import argparse
+import logging
+
+
+def main(args: argparse.Namespace) -> None:
+    with open(args.gold, "r") as gf, open(args.pred, "r") as pf:
+        with open(args.out, "w") as wf:
+            for i, (g_line, p_line) in enumerate(zip(gf, pf)):
+                try:
```

Comment:
This try/assert/except AssertionError design is a weird one for me. Instead of that, I'd do an if-statement, log the warning, and continue.